docs: Reorganize user guide (#2050)
webrecorder/browsertrix: docs/user-guide/running-crawl.md

> # Modifying running crawls

Reviewer: ```suggestion
# Modifying Running Crawls
```

> ## In Progress Status

Reviewer: ```suggestion
## In-Progress Statuses
```

> A crawl workflow that is in progress can be in one of five possible states:

Reviewer 2: ```suggestion
A crawl workflow that is in progress can be in one of six possible states:
```

Must have added one at some point :)

Author: Do we need to state the number? "A crawl workflow that is in progress can be in one of the following states:" or similar also gets the point across

Reviewer 2: ```suggestion
A crawl workflow that is in progress can be in one of the following states:
```

Good point!
